argoproj-crd/Makefile

```diff
@@ -1,18 +1,16 @@
-.PHONY: apply
-apply:
-	@if ! kubectl get namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)" >/dev/null 2>&1; then \
-		kubectl create namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)"; \
-	fi
-	helm repo add argo https://argoproj.github.io/argo-helm;
-	helm --kube-context $${KUBE_CONTEXT:-"dind"} --tiller-namespace $${TILLER_NAMESPACE:-"kube-system"} upgrade --wait --install --namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)" $$(basename $$PWD)-$$(git rev-parse --short HEAD)-argo-cd argo/argo-cd;
-	helm --kube-context $${KUBE_CONTEXT:-"dind"} --tiller-namespace $${TILLER_NAMESPACE:-"kube-system"} upgrade --wait --install --namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)" -f ci.yaml $$(basename $$PWD)-$$(git rev-parse --short HEAD) .;
+RELEASE = "$$(basename $$PWD)"
+
+.PHONY: install
+install:
+	helm repo add argo https://argoproj.github.io/argo-helm
+	helm upgrade -i --wait $(RELEASE)-argo-cd argo/argo-cd
```

Comment:
Does it fixed namespace `default` ?

Reply:
@cw-sakamoto 
No, always install against the current namespace.

Reply:
Which is `current namespace` ?
I think it will be `default` if you don't create it and don't specify it.

But, first of all, I don't know if there is a need to specify namespace. Why did we specify it ..?🤔

Reply:
@cw-sakamoto 
You need to think about namespace in terms of CI and local.

In CI, we create and use namespace for the application name.
https://github.com/chatwork/charts/pull/157/files#diff-1d37e48f9ceff6d8030570cd36286a61R14-R16

Locally, we've been using namespaces that use a committed hash.
https://github.com/chatwork/charts/pull/157/files#diff-3b201ba249700e0681cda702f73e30e9L3-L5
However, this was causing inconvenience and complexity in Makefile when developing locally.


So I decided to change the namespace not to be specified in Makefile.

Reply:
Oh, sorry.
 I don't confirm base setting. 😱